<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KCMWacomTabletWidget</class>
 <widget class="QWidget" name="KCMWacomTabletWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphic Tablet Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="tabletListLabel">
       <property name="text">
        <string>&amp;Select Tablet:</string>
       </property>
       <property name="buddy">
        <cstring>tabletListSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="tabletListSelector">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="profileLabel">
       <property name="text">
        <string>Select &amp;Profile:</string>
       </property>
       <property name="buddy">
        <cstring>profileSelector</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="profileLayout">
       <item>
        <widget class="QComboBox" name="profileSelector">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KAnimatedButton" name="addProfileButton">
         <property name="toolTip">
          <string>Creates a new default profile</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
         <property name="animationPath" stdset="0">
          <string notr="true">document-new</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KAnimatedButton" name="delProfileButton">
         <property name="toolTip">
          <string>Deletes the current profile</string>
         </property>
         <property name="animationPath" stdset="0">
          <string notr="true">edit-delete-page</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="deviceTabWidget">
     <property name="documentMode">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KAnimatedButton</class>
   <extends>QToolButton</extends>
   <header>kanimatedbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>profileSelector</tabstop>
  <tabstop>addProfileButton</tabstop>
  <tabstop>delProfileButton</tabstop>
  <tabstop>deviceTabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
